Plattform/package.json

```diff
@@ -8,6 +8,7 @@
         "build": "gatsby build",
         "deploy": "firebase deploy --only hosting",
         "develop": "gatsby develop",
+        "gatsbyDebug": "node --nolazy node_modules/.bin/gatsby develop --inspect-brk",
```

Comment:
als goody hab ich hier noch ein script eingefügt, das uns beim debugging von gatsby unterstützt. Vorraussetzung sind VS code settings, und der Start über das integrierte Terminal:
```
 "debug.node.autoAttach": "on",
```

Reply:
https://www.gatsbyjs.com/docs/debugging-the-build-process/#vs-code-debugger-auto-config